php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S2681.html

```diff
@@ -9,6 +9,11 @@
   secondAction();  // Noncompliant; executed unconditionally
 thirdAction();
 
+if($condition) firstActionInBlock(); secondAction();  // Noncompliant; secondAction executed unconditionally
+
+if($condition) firstActionInBlock();  // Noncompliant
+  secondAction();  // Executed unconditionally
+
```

Comment:
implementation do not support those cases

Reply:
Indeed, will be handled by [SONARPHP-695](https://jira.sonarsource.com/browse/SONARPHP-695).